docs/UserGuide.md

```diff
@@ -662,11 +735,11 @@ Warenager will suggest:
 list lt/all
 ```
 
-### Generates a csv file that contains all stocks: `print`
-Generates a csv file that contains all stocks. Csv file will be named according to the user input, and the file name
+### Generates a CSV file that contains all stocks: `print`
+Generates a CSV file that contains all stocks. Csv file will be named according to the user input, and the file name
 can only contain alphanumeric characters. Users may want to sort the stocks using `sort` command
-to sort the stock in their preferred order before converting it into the csv file. The csv file is saved
-to `[root directory]/data/userInput.csv` after successfully executing the command.
+to sort the stock in their preferred order before converting it into the CSV file. The CSV file is saved
+to `[root directory]/data/userInput.CSV` after successfully executing the command.
```

Comment:
I think the `CSV` on `userInput.CSV` should be lowercased as in this context it is a file extension. Same issue with other `CSV` below.

Reply:
Fixed!